### PR TITLE
Dev update 1 46 0

### PR DESCRIPTION
**Pull request by mbeacom**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-  "name": "cdk-watchful",
-  "version": "0.5.0",
-  "description": "Watching your CDK apps since 2019",
+  "name": "cdk-watchful-edge",
+  "version": "0.6.0",
+  "description": "Watching your CDK apps since 2019 - Edge",
```
Comment (eladb) [inline package.json:4]: Why the name change?

Comment (mbeacom) [reply package.json:4]: @eladb Apologies for the erroneous PR!  I was attempting to settle the versioning issues with the latest release of CDK without impacting the official release workflow.  If you're open to a PR to update to 1.46.0, I'll open one without the name change.

Comment (eladb) [reply package.json:4]: Absolutely

Comment (mbeacom) [reply package.json:4]: @eladb https://github.com/eladb/cdk-watchful/pull/20

**Pull request closed without merging**